### DISCUSSION TITLE
Integrate a pre-written FastAPI to Pynecone

### DISCUSSION THREAD

**Discussion by ramsy0dev** [reflex-dev/reflex#789] [Q&A]
I have made a FastAPI API and i wonder if i can integrate it to Pynecone directly or should i just run it elsewhere

**Comment by rgimen3z** [comment]
Any documentation on integrating with an existing FastAPI service updated as of 2025?

**Reply by libardoram** [reply]
Take a look [https://reflex.dev/docs/api-routes/overview/](https://reflex.dev/docs/api-routes/overview/), it helps me with a couple of api routes I needed to move.
